src/styles/blocks/nav.scss

```
@@ -0,0 +1,49 @@
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  margin-left: 20px;
```

```suggestion
.nav {
  &__list {
    display: flex;
    align-items: center;
  }

  &__item {
    margin-left: 20px;
```
Nesting can be used here as well